docs/Local Development Install.md

```diff
@@ -0,0 +1,181 @@
+Table of Contents
+
+1. [DCMFOSS Local Development Deployment](#dcmfoss-local-development-deployment)
+   - [Prerequisites](#prerequisites)
+   - [Guide](#guide)
+      - [Cloning the repository](#cloning-the-repository)
+      - [Open in your own IDE of choice](#open-in-your-own-ide-of-choice)
+      - [Docker desktop](#docker-desktop)
+      - [Running containers](#running-containers)
+      - [Running project first time for init configs](#running-project-first-time-for-init-configs)
+      - [Running Keycloak](#running-keycloak)
+      - [Fetching the keycloak client credential](#fetching-the-keycloak-client-credential)
+      - [Configure postman collection requests](#configure-postman-collection-requests)
+      - [Run the front-end](#run-the-front-end)
+   - [Postmand Collection](#Postman-collection)
+   - [Notice](#notice)
+
+
+# DCMFOSS Local Development Deployment
+
+
+## Prerequisites
+
+This guide assumes the following prerequisites are true.
+It was built considering an updated windows build with docker running, but will try to provide general instructions.
+
+  - Basic coding knowledge
+  - Basic Docker/Kubernetes knowledge
+  - JDK 17 ( or Higher)
+  - Maven 'Spring Boot is compatible with Apache Maven 3.5 or above'
+  - Git/Github Desktop
+  - NodeJS installed
+  - Yarn installed
+  - Docker Desktop
+
+
+## Guide:
+Let's begin local development install!
+
+- ### Cloning the repository
+    Clone the repository from the url below
+
+      git clone https://github.com/catenax-ng/tx-demand-capacity-mgmt.git
+  
+    ![Using Github desktop](images/dev/1.png "Cloning the repo")
+
+- ### Open in your own IDE of choice
+    Boot your IDE of choice and open the backend repo,
+    add the specification as a module of the backend project.
+    
+    ![IntelliJ IDE](images/dev/2.png "resolving dependencies")
+
+    ![IntelliJ IDE modules](images/dev/2_5.png "resolving dependencies")
+
+  it is normal for startup to have errors, let your IDE finish configuring and download dependencies.
+
+  go ahead and run a maven clean and install on the specification cycle.
+  if it fails saying you don't have a JDK, choose JDK17 of your choice.
+
+- ### Docker desktop
```

Comment:
Consider making this part more generic. 
- Docker Desktop is a non-open source solution
- When Docker is a pre-requisite, just point to the official install docs. They provide OS specific instructions
- Also WSL instructions are far better described by the WSL project itself

Reply:
Fixed via commit:
https://github.com/catenax-ng/tx-demand-capacity-mgmt/commit/493d1f19f2691ba781a094af4a0efbbc3214f6a8